lib/model/message_list.dart

```diff
@@ -381,6 +381,7 @@ class MessageListView with ChangeNotifier, _MessageSequence {
     for (final message in result.messages) {
       if (_messageVisible(message)) {
         _addMessage(message);
+        store.recentSenders.processStreamMessage(message);
```

Comment:
Can you talk a bit about the performance considerations of this?

Reply:
I am not hundred percent sure if it's an expensive task or not. I think it does relatively simple task of adding `IdTracker` to a map two times.